index.html

```diff
@@ -89,27 +97,110 @@
 <script>
   var data = [];
   var options = {
-    yearRange: 'all'
+    SOCGroup: 'all'
   };
 
-  var svg;
+  var svg, x, y;
+
+  var margin = { top: 15, right: 35, bottom: 30, left: 50};
+
+  var width=640 - margin.right - margin.left;
+  var height=400 - margin.top - margin.bottom;
 
   // FETCH DATA
 
+  d3.json('data.json', function(error, json){
+    if (error) { throw error; }
+
+    data=json;
+
+    setup();
+    update();
+  });
+
   function setup() {
     // SVG
 
     svg = d3.select('#chart')
-      .append('svg');
-
-    // SCALES
-
+      .append('svg')
+      .attr('width', width + margin.right + margin.left)
+      .attr('height', height + margin.top + margin.bottom)
+      .append('g')
+      .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
+
+    // SCALES, domain min to max (which ends up at width)
+
+    x = d3.scale.linear()
+      .domain([0, d3.max(data, function (d) {return d.WF_Female;})])
+      .range([0, width])
+      .nice();
+
+    y = d3.scale.linear()
+      .domain(d3.extent(data, function (d) {return d.MedEarn; }))
+      .range([height, 0])
+      .nice();
+    
     // AXES
-
+    var formatAsPercentage = d3.format(".0%");
+
+    var xAxis = d3.svg.axis()
+                  .scale(x)
+                  .tickFormat(formatAsPercentage)
+                  .tickSize(height)
+                  .orient("bottom");
+
+    svg.append("g")
+        .attr("class", "x axis")
+        .call(xAxis)
+      .append("text")
+        .attr("x", 200)
+        .attr("y", 370)
+        .attr("dy", ".71em")
+        .style("text-anchor", "start")
+        .text("Percent of Workers that are Women");
+
+      var gx = svg.append("g")
+          .attr("class", "x axis")
+          .call(xAxis);
+
+      gx.selectAll("g").filter(function(d) { return d; })
+          .classed("minor", true);
+
+
+    var formatAsDollars = d3.format("$1,000")
+
+    var yAxis = d3.svg.axis()
```

Comment:
I would like to get rid of the y-axis labels on the right hand side, but I'm not seeing where they are created so I can remove them.